src/main/java/core/basesyntax/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.impl.FruitShopServiceImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+
+    @Override
+    public void processWithTransaction(FruitTransaction transaction) {
+        FruitShopService fruitShopService = new FruitShopServiceImpl();
+        fruitShopService.balanceOfFruit(transaction);
```

Comment:
Don't see sense to create additional service to just call it's method from handler.
Just move logic to handlers

Reply:
not fixed